tokio/src/runtime/task/raw.rs

```diff
@@ -188,15 +163,11 @@ const fn get_id_offset(
 /// Pseudo-code for the `#[repr(C)]` algorithm can be found here:
 /// <https://doc.rust-lang.org/reference/type-layout.html#reprc-structs>
 #[cfg(tokio_unstable)]
-const fn get_spawn_location_offset(
-    header_size: usize,
-    core_align: usize,
-    scheduler_size: usize,
-    id_align: usize,
-    id_size: usize,
-    spawn_location_align: usize,
-) -> usize {
-    let mut offset = get_id_offset(header_size, core_align, scheduler_size, id_align);
+const fn get_spawn_location_offset<T1, T2, T3, T4, T5>() -> usize {
```

Comment:
The types are literally just T1, T2, T3, T4, T5. This change makes readability of this code worse.